x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/DoubleBigArrayBlock.java

```diff
@@ -36,20 +36,18 @@ public DoubleBigArrayBlock(
             positionCount,
             firstValueIndexes,
             nulls,
-            mvOrdering,
-            blockFactory
+            mvOrdering
         );
     }
 
     private DoubleBigArrayBlock(
-        DoubleBigArrayVector vector,
+        DoubleBigArrayVector vector, // stylecheck
```

Comment:
What's with the //stylecheck?

Reply:
Ah, our style check enforces a single-line format for short type names like 'Int' and a multiple-line format for longer type names such as 'Boolean' or 'BytesRef' for this constructor. I've added this comment to bypass the style check.